template_feed/Microsoft.DotNet.Common.ItemTemplates/content/Gitignore/.template.config/template.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [
+    "Config"
+  ],
+  "name": "dotnet gitignore file",
+  "generatorVersions": "[1.0.0.0-*)",
+  "description": "Creates an empty gitignore file.",
```

Comment:
"Empty" should probably be changed here.

Reply:
to "Creates an Empty gitignore file." ?

Reply:
Maybe "Creates a gitignore file for a dotnet project".?

Reply:
let's continue in email, cannot find all people in github

Reply:
Changed to "Creates a gitignore file for a dotnet project.”